ntoskrnl/mm/ARM3/virtual.c

```diff
@@ -719,36 +710,26 @@ MiDeleteVirtualAddresses(IN ULONG_PTR Va,
                     /* The PTE was never mapped, just nuke it here */
                     MI_ERASE_PTE(PointerPte);
                 }
+
+                if (MiDecrementPageTableReferences((PVOID)Va) == 0)
+                {
+                    ASSERT(PointerPde->u.Long != 0);
+                    ASSERT(MiIsPteOnPdeBoundary(PointerPte + 1) || ((Va + PAGE_SIZE) > EndingAddress));
```

Comment:
This fails on x64. And it doesn't seem correct. There can be 0 PTEs at the end of a page table.